invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/Identifiers/PIDField/components/OptionalDOIoptions.js

```diff
@@ -36,19 +37,25 @@ class OptionalDOIoptionsCmp extends Component {
     } = this.props;
 
     const doi = record?.pids?.doi?.identifier || "";
+    const isInternalProvider = record?.pids?.doi?.provider !== PROVIDER_EXTERNAL;
```

Comment:
will this condition also be true when we have no DOI? (No I don't need one option)